### ISSUE TITLE
Suggestion: apt repo

### ISSUE THREAD

**Issue by sbrl** [timvisee/ffsend#45]
An `apt` repository for those using a Debian-base distribution who don't use `snap` would be great.

Since packages are already been generated for other platforms, perhaps a tool like [fpm](https://fpm.readthedocs.io/en/latest/packages.html) could be used to convert 1 to the other?

[Guides exist](https://medium.com/sqooba/create-your-own-custom-and-authenticated-apt-repository-1e4a4cf0b864) to walk through the setting up of an apt repo too, though of course the apache server doesn't have to be used - another or a pre-existing server can be used instead.

**Comment by timvisee** [comment]
Yes, I like this. I primarily use Debian/Ubuntu based systems myself and like real packages better in contrast to `snap`.

About half a year ago I attempted to build a `.deb` package, and wanted to publish it to my personal PPA. I didn't have clear instructions on how to properly do that though with a Rust binary, and eventually stopped attempting it because I had to focus on other things. I do have a script [available](https://github.com/timvisee/ffsend/blob/master/pkg/create_deb) to generate a `.deb` package, but actually publishing the source version to the PPA would be the next step.

I don't have time to look into this right now, but I definitely want to implement and release this as soon as possible. Thank you for linking the guide, it seems interesting. It might help me realize this.

**Comment by aurora-of-earth** [comment]
@timvisee 
Please release packages on the Debian's official repos.

**Comment by sbrl** [comment]
@aurora-of-earth Not sure whether that'll happen, since getting stuff into mainline Debian is really hard & takes ages. An apt repo is a *much* quicker solution

**Comment by m3thm4th** [comment]
PPA is Ubuntu's exclusive and requires to be set up to work (poorly) on Debian. A deb repo is better than PPA being simpler, faster and better integrated in the system.

**Comment by sbrl** [comment]
Ah, that's just my poor choice of wording there @m3thm4th haha

**Comment by werdahias** [comment]
fwiw this is kinda blocked by https://github.com/timvisee/ffsend-api/issues/68 ; I prepared all other dependencies for debian. Introducing a deprecated package (websockets) is not a good idea; unfortunately this blocks a debian package.
